autospec/buildreq.py

```diff
@@ -825,16 +825,17 @@ def scan_for_configure(dirn, tname, dlpath):
                 # so only use go directly if we can't find a Makefile
                 buildpattern.set_build_pattern("golang", default_score)
```

Comment:
Does this build pattern setting still make sense if we are checking for `golang-mod` below?

Reply:
I think in those cases we'd likely need to figure out a mechanism to build the package (but usually it is create a patch for the Makefile). For now it is probably the most sensible option but I'm open to suggestions.

Reply:
Okay, let's leave it as-is for now.